no function or associated item named `from_os_rng` found for struct `StdRng`

**Describe the bug**
error[E0599]: no function or associated item named `from_os_rng` found for struct `StdRng` in the current scope
   --> D:\rust\cargo\registry\src\mirrors.ustc.edu.cn-5857e57f01837ef8\burn-dataset-0.19.1\src\transform\options.rs:48:43
    |
 48 |             RngSource::Default => StdRng::from_os_rng(),
    |                                           ^^^^^^^^^^^ function or associated item not found in `StdRng`

**To Reproduce**
run wasm-pack build
```
...
[lib]
crate-type = ["cdylib", "rlib"]

[features]
default = ["console_error_panic_hook","ndarray"]
ndarray = ["burn/ndarray"]
wgpu = ["burn/wgpu"]



[dependencies]
wasm-bindgen = "0.2.84"
wasm-bindgen-futures = "0.4.56"
js-sys = "0.3.83"
# https://github.com/rust-random/getrandom/#webassembly-support
getrandom = { version = "0.3.4", default-features = false, features = [
    "wasm_js",
] }
burn = { version = "0.19.1", features = ["dataset"],default-features = false }

# The `console_error_panic_hook` crate provides better debugging of panics by
# logging them with `console.error`. This is great for development, but requires
# all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
# code size when deploying.
console_error_panic_hook = { version = "0.1.7", optional = true }


[dev-dependencies]
wasm-bindgen-test = "0.3.34"

[profile.release]
# Tell `rustc` to optimize for small code size.
opt-level = "s"
```

**Expected behavior**
success run wasm-pack build


**Desktop (please complete the following information):**
 - OS: windows11 
 - Browser None
 - Version  0.19.1

--- a/crates/burn-dataset/Cargo.toml
+++ b/crates/burn-dataset/Cargo.toml
@@ -57,7 +57,7 @@ encoding_rs = { workspace = true, optional = true }
 polars = { workspace = true, optional = true }
 r2d2 = { workspace = true, optional = true }
 r2d2_sqlite = { workspace = true, optional = true }
-rand = { workspace = true, features = ["std"] }
+rand = { workspace = true, features = ["std", "os_rng"] }
 zip = { workspace = true, optional = true }
 rmp-serde = { workspace = true }
 rusqlite = { workspace = true, optional = true }

